app/mailers/thing_mailer.rb

```diff
@@ -1,9 +1,15 @@
 class ThingMailer < ActionMailer::Base
   default from: 'adoptadrain@codeforsanfrancisco.org'
 
+  def adopted_confirmation(thing)
+    @thing = thing
+    @user = thing.user
+    mail(to: @user.email, subject: ['Thank you for Adopting a Drain in San Francico'])
```

Comment:
Not a big deal, but it seems like we have a lot of this sort of text as locale specific translations (i.e. https://github.com/sfbrigade/adopt-a-drain/blob/production/config/locales/en.yml and friends).

I don't know if we actually plan on doing i18n with this -- I could conceivably see it translated to other languages (e.g. spanish), but I'm not sure who would do the translating or if we would just rely on Google Translate.